projects/media-viewer/src/lib/store/store-utils.ts

```diff
@@ -26,6 +27,52 @@ export class StoreUtils {
       }, {});
   }
 
+  static genTagNameEntities(annotations) {
+    const filterAnnoWithNoCommentsTags = annotations.filter(a => (a.comments.length && a.tags.length));
```

Comment:
this filter gives you annotations that have comments and tags, so the name is misleading

Reply:
why don't u suggest something then